### DISCUSSION TITLE
E-Mail Marketing/Templates

### DISCUSSION THREAD

**Discussion by rubenmarcus** [frontendbr/forum#160] [Discussão]
Oi quero comentar e perguntar pra outros devs como vocês fazem os templates de e-mail de clientes e como testam?

**Testes:** Fazem apenas pra web ou pra aplicativos antigos?
**Modelar e boas práticas:** Usam muitas imagens?Poucas? Nenhuma? Modelam nos apps de e-mail(Outlook,Thunderbird) ou IDES?

**Como Evitar cair na caixa de SPAM?**

No google quando se procura compatibilidade e boas práticas, você acha facil algumas dessas infos.. Inclusive uma página da MailChimp muito boa.

Mas queria perguntar pra quem faz tambem, como é o fluxo de desenvolvimento.

**Comment by mksanti** [comment]
uma ferramenta que utilizo nesses casos.
https://mjml.io/

**Comment by bomffas** [comment]
No trampo geralmente os emails são todos compostos por imagens, então eu faço os cortes com slices no photoshop e exporto em html, fazendo alguns ajustes com css inline:

Nas `<tr>, <td>` adiciono:

``` html
style="margin:0;padding:0;"
```

e nas imagens:

``` html
style="margin:0;padding:0;border:0;display:block;"
```

e na `<table>` adiciono o alinhamento ao centro:

``` html
align="center"
```

É bom evitar links para arquivos como `.rar, .exe, .zip` pois caem no spam, como também caem assuntos das mensagem contendo palavras como: **Compra, Venda, Clique Aqui, Grátis**.

Bom também é transcrever os emails em formato de texto, assim se o cliente de e-mail não puder abrir a imagem a pessoa ainda consegue ler do que se trata o e-mail.

**Comment by felipefialho** [comment]
Hoje em dia é raro eu fazer newsletters, mas eu tinha criado um [boilerplate](https://github.com/LFeh/newsletter-boilerplate) meio que a prova de bugs (na maioria dos clientes de email da época).

Talvez ainda seja uma boa base.

**Comment by onaSousa** [comment]
poha essa ferramenta @mikejavier muito foda!

para montar uso o Builder do Litmus
no Mailchimp eu testo com a ferramenta do Litmus como fica a visualizao nos principais clientes de email
evito usar imagens mas isso eh opcao minha msm jah que sou eu quem faz o layout, prefiro focar em uma fonte legal mas claro que nem sempre eh possivel heheh
ah e sigo muito do que vejo nos artigos do Litmus e na comunidade tbm
